patches/minecraft/net/minecraft/client/renderer/entity/RenderLivingBase.java.patch

```diff
@@ -33,15 +33,25 @@
              {
                  EntityLivingBase entitylivingbase = (EntityLivingBase)p_76986_1_.func_184187_bx();
                  f = this.func_77034_a(entitylivingbase.field_70760_ar, entitylivingbase.field_70761_aq, p_76986_9_);
-@@ -192,6 +197,7 @@
+@@ -107,6 +112,9 @@
+                 {
+                     f += f3 * 0.2F;
+                 }
++                // Forge: Fix MC-1207
++                f2 = f1 - f;
++
```

Comment:
Please reduce this patch to 1 line total instead of 3:
`f2 = f1 - f; // Forge: Fix MC-1207`